apollo/src/graphql/user/userQueries.ts

```diff
@@ -15,4 +17,21 @@ builder.queryFields((t) => ({
             return apiKeys;
         },
     }),
+    listUsers: t.field({
+        type: [User],
+        authScopes: {
+            loggedIn: true,
+        },
+        args: {
+            teamId: t.arg.string({ required: true }),
+        },
+        async resolve(_root, args, ctx) {
+            await ObjectionTeamEdge.userHasRole(ctx.user.$id(), args.teamId, ['owner']);
+
+            const users = await ObjectionUser.query()
+                .where('isApproved', '=', true)
```

Comment:
This might also be something we don't need? If we make this a more generic method, as an administrator there's utility in being able to see if a user is or isn't verified so I can hound them to click the damn email link. It might make more sense to just add an `isVerified` modifier on ObjectionUser and add that to the query builder when we hit the mutation to add a user to a team

Reply:
Update to add, modifiers: https://vincit.github.io/objection.js/recipes/modifiers.html

IDK if that's a corner of Objection you e poked around before

Reply:
There's many corners of Objection I have not poked around lol